packages/zosfiles/src/methods/copy/Copy.ts

```diff
@@ -139,10 +161,7 @@ export class Copy {
      */
 
     public static async copyPDS (
-        session: AbstractSession,
-        fromPds: string,
-        toPds: string
-    ): Promise<IZosFilesResponse> {
+        session: AbstractSession, fromPds: string, toPds: string, newDataSet: boolean): Promise<IZosFilesResponse> {
```

Comment:
Since `copyPDS` is a public method we can't make breaking changes to the function signature. If we add a parameter like `newDataSet` it needs to be optional:
```suggestion
        session: AbstractSession, fromPds: string, toPds: string, newDataSet = false): Promise<IZosFilesResponse> {
```